wordle/src/main/kotlin/wordle/domain/FixedAnswerExtractor.kt

```diff
@@ -0,0 +1,19 @@
+package wordle.domain
+
+import java.time.Duration
+import java.time.LocalDateTime
+import kotlin.math.abs
+
+class FixedAnswerExtractor : AnswerExtractor {
+
+    override fun extract(words: List<Word>): Word {
+        val index = abs(toDays() % words.size)
```

Comment:
나머지 연산에 절댓값을 씌우는 특별한 이유가 있을까요? 아래와 같이 제거해도 테스트는 모두 통과하는 것으로 보여요.

```suggestion
        val index = toDays() % words.size
```

Reply:
어... 안되길래 절대값 연산을 했던 것 같은데, 확인해보고 불필요하면 제거하겠습니다.